web/beacon-app/src/features/projects/types/projectQueryService.ts

```diff
@@ -14,3 +14,21 @@ export interface ProjectQueryDTO {
   projectID: string;
   query: string;
 }
+
+export interface MetaDataResponse {
+  key: string;
```

Comment:
The response will be like `"key": "value"`. Would that require an update to the `MetaDataResponse`?

Reply:
Absolutely. I just working on setting up the MetaData type definition to align with the upcoming story that will handle the data appropriately.